packages/roosterjs-plugin-image-resize/lib/ImageResize.ts

```diff
@@ -39,12 +39,14 @@ export default class ImageResize implements EditorPlugin {
      * @param minHeight Minimum height of image when resize in pixel, default value is 10
      * @param selectionBorderColor Color of resize border and handles, default value is #DB626C
      * @param forcePreserveRatio Whether always preserve width/height ratio when resize, default value is false
+     * @param bypassClassName Class name of IMG tag which to bypass resize logic (e.g. for a placeholder image)
      */
     constructor(
         private minWidth: number = 10,
         private minHeight: number = 10,
         private selectionBorderColor: string = '#DB626C',
-        private forcePreserveRatio: boolean = false
+        private forcePreserveRatio: boolean = false,
+        private bypassClassName: string = undefined
```

Comment:
Instead of having a bypass class name, I'd like to make it more common. Suggest to pass in a "resizableImageSelector" with default value "img". So that people can use any selector to determine their image to resize.